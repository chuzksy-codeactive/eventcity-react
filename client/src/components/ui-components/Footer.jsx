import React from 'react';

/**
 * This component is used for the footer
 * of the app
 *
 * @function 
 * @return {object} JSX DOM
 */
const Footer = () => {
  return (
    <footer className="footer">
      <div className="container ">
        <p className="m-0 text-center text-white ">Copyright &copy; EventCity 2018</p>
      </div>
    </footer>
  );
};

export default Footer;
