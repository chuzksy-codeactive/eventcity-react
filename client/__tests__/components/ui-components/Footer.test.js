import React from 'react'
import Footer from '../../../src/components/ui-components/Footer'

describe('Footer Component test', () => {
  it('It should render the component without error', () => {
    const wrapper = renderer.create(<Footer />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
})