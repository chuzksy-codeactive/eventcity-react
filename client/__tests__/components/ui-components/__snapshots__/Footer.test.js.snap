// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component test It should render the component without error 1`] = `
<footer
  className="footer"
>
  <div
    className="container "
  >
    <p
      className="m-0 text-center text-white "
    >
      Copyright Â© EventCity 2018
    </p>
  </div>
</footer>
`;
